composer.json

```diff
@@ -18,13 +18,16 @@
 		"source": "https://github.com/jweiland-net/weather2"
 	},
 	"require": {
-		"typo3/cms-core": "^10.4.29 || ^11.5.11",
-		"sjbr/static-info-tables": ">=6.6.0"
+		"typo3/cms-core": "^12.4",
+		"typo3/cms-dashboard": "^12.4",
```

Comment:
I can't find a dashboard widget in Services.yaml. Why EXT:dashboard is needed?

Reply:
Its imported at the top of the file using 
`imports:
  - { resource: Backend/DashboardWidgets.yaml }`